scripts-dev/release.py

```diff
@@ -334,7 +334,7 @@ def publish(gh_token: Optional[str]):
         return
 
     if gh_token:
```

Comment:
given that this now does nothing without a gh_token, I think it should error out rather than fail silently.